addons/crm_mail_client_extension/views/crm_lead_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+    <!-- Called by the plugin to open a lead in edit mode -->
+        <record id="action_lead_edit_mode" model="ir.actions.act_window">
+              <field name="name">Redirection to the lead edit form</field>
```

Comment:
We try to namespace names if possible, like "Lead: redirect to form view in edit mode".